schema.md

```diff
@@ -0,0 +1,555 @@
+
+* Type: string or array of strings
+* Presence: optional
+* Example: "https://github.com/italia/otello"
+
+This string must be one or multiple URLs of upstream repositories from
+which this fork was created. This key is meant to create a relation
+between forked repositories that could not be found out otherwise (for
+instance, some code hosting platforms record fork relations in their
+metadata).
+
+If you fork a software with the purpose of creating an independent
+version, you should consider mentioning the upstream project in this
+key.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `legal`
+
+### Key `legal/license`
+
+* Type: string
+* Presence: mandatory
+* Example: `"AGPL-3.0-or-later"`
+
+This string describes the license under which the software is
+distributed. The string must contain a valid SPDX expression, referring
+to one (or multiple) open-source license. Please refer to the [SPDX
+documentation](https://spdx.org/licenses/) for further information.
+
+### Key `legal/main-copyright-owner`
+
+* Type: string
+* Presence: optional
+* Example: "City of Amsterdam and many contributors"
+
+This string describes the entity that owns the copyright on
+"most" of the code in the repository. Normally, this is the line
+that is reported with the copyright symbol at the top of most files
+in the repo.
+
+It is possible to list multiple owners if required so, using an English
+sentence. It is also possible to informally refer to a community of
+group of people like "Linus Torvalds and all Linux contributors".
+
+In case it is not possible to name a main copyright owner, it is
+possible to omit this key; in those cases, if the repo has a authors
+file, you can point to it through `legal/authors_file`.
+
+### Key `legal/authors-file`
+
+* Type: string (path to file)
+* Presence: optional
+* Example: "doc/AUTHORS.txt"
+
+Some open-source softwares adopt a convention of identify the copyright
+holders through a file that lists all the entities that own the
+copyright. This is common in projects strongly backed by a community
+where there are many external contributors and no clear single/main
+copyright owner. In such cases, this key can be used to refer to the
+authors file, using a path relative to the root of the repository.
+
+### Key `legal/repo-owner`
+
+* Type: string
+* Presence: mandatory
+* Example: "City of Amsterday"
+
+This string describes the entity that owns this repository; this might
+or might not be the same entity who owns the copyright on the code
+itself. For instance, in case of a fork of the original software, the
+`repo-owner` is probably different from the `main-copyright-owner`.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `maintenance`
+
+This section provides information on the maintenance status of the
+software, useful to evaluate whether the software is actively
+developed or not.
+
+### Key `maintenance/type`
+
+* Type: enumerate
+* Presence: mandatory
+* Values: "commercial", "community", "none"
```

Comment:
Can I suggest:

Values:
* `internal`: maintenance is performed by the authors
* `contract`: maintenance is performed by a contracted party
* `community`: maintenance is done by the community
* `none`: this project is unmaintained, if you file a pull request you'll likely get no response

Reply:
Good idea.